lib/admin/WP_Auth0_Admin.php

```diff
@@ -49,42 +40,26 @@ public function admin_enqueue() {
 				'ajax_url'                => admin_url( 'admin-ajax.php' ),
 			]
 		);
-		wp_register_script( 'wpa0_async', WPA0_PLUGIN_LIB_URL . 'async.min.js', false, WPA0_VERSION );
 
 		$wpa0_pages     = [ 'wpa0', 'wpa0-errors', 'wpa0-import-settings', 'wpa0-setup' ];
 		$wpa0_curr_page = ! empty( $_REQUEST['page'] ) ? $_REQUEST['page'] : '';
 		if ( ! in_array( $wpa0_curr_page, $wpa0_pages ) ) {
-			return;
-		}
-
-		if ( ! WP_Auth0::ready() && 'wpa0-setup' !== $_REQUEST['page'] ) {
-			add_action( 'admin_notices', [ $this, 'create_account_message' ] );
+			return false;
```

Comment:
What does this false signify?  It used to be just return and the end of method also does not return a boolean.

Reply:
Nothing in a functional sense. This method is ran at a certain time during execution and its return value is unused. I'm now using it for testing purposes.